src/facades/ReleaseCommunicationFacade.js

```diff
@@ -5,19 +5,21 @@ const {
   getTagsHandler,
   getTagDiffHandler,
   postMessageHandler,
-  searchIssuesByCommitHandler,
+  searchIssuesByCommitHandler
 } = require('../handlers');
 const { getTagDiffFromTagId, ticketFinder } = require('../utils');
 
 const PR_TEMPLATE_COMMENT_REGEX = new RegExp(/<!--[\s\S]*?-->/, 'gm');
 const STRIP_IGNORE_TICKETS = new RegExp(/<!--.+?icl.+?-->([\S\s.]*?)<!--.+?ecl.+?-->/, 'gm');
+const STRIP_IGNORE_TICKETS_LONG = new RegExp(
+  /<!--.+?captains-log-ignore.+?-->([\S\s.]*?)<!--.+?end-captains-log-ignore.+?-->/,
```

Comment:
Usually I use either the `/abc/gm` literal regex or the `new RegExp('abc', 'gm')` constructor but I've not really used the `RegExp` constructor with a regex literal as the argument - are we sure this works as expected?

Should we really match any character between `<!--` and `captains-log-ignore` or only inline whitespace? I guess maybe newlines too? Could try `[\n\r\s]+`? Otherwise we might accidentally match a comment like this:
```
<!-- Here we will use captains-log-ignore to not report to captains-log -->
```

Are we sure the `.` inside `[\S\s.]` matches any character and is needed? Try `/[.]/.test('a')` - it's `false` - it only matches literal `.` (`/[.]/.test('.')` is `true`). But the `\S\s` is enough to match any character so I guess it works. 🤷‍♂ Alternatively, we may be able to use the `/s` modifier ([`dotAll`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/dotAll)) which causes `.` to match newlines too - but it's only supported in some JS environments like Node 8.10+ - do we support any lower than that?

We could try this regex?

```
const STRIP_IGNORE_TICKETS_LONG =
  /<!--[\n\r\s]+?captains-log-ignore[\n\r\s]+?-->.*?<!--[\n\r\s]+?end-captains-log-ignore[\n\r\s]+?-->/gms;
```

Reply:
Let's follow up on this in a subsequent PR. I think I can work with you to make these regexs a bit more accurate. Thank you for the detailed reply here! I'll create an issue with this content in it.

Reply:
https://github.com/target/captains-log/issues/58